semantic-conventions/src/tests/semconv/model/test_correct_parse.py

```diff
@@ -669,6 +669,23 @@ def semantic_convention_check(self, s, expected):
         self.assertEqual(expected["n_constraints"], len(s.constraints))
         self.assertEqual(expected["attributes"], [a.fqn for a in s.attributes])
 
+    def test_scope_attribute(self):
+        semconv = SemanticConventionSet(debug=False)
+        semconv.parse(self.load_file("yaml/scope.yaml"))
+        self.assertEqual(len(semconv.models), 1)
+
+        expected = {
+            "id": "scope",
+            "prefix": "scope",
```

Comment:
I think you should also assert for the `type` to be `scope` here.
Ideally also use some dummy names for id and prefix - if it's always `scope` in all occasions the test would not be able to discover any mix-up of the different fields.

Reply:
done